src/FFNetAgent.cpp

```diff
@@ -0,0 +1,365 @@
+/*
+ * Copyright (c) 2015 - 2023, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "FFNetAgent.hpp"
+
+#include <sstream>
+#include <cmath>
+#include <cassert>
+#include <algorithm>
+#include <fstream>
+
+#include "PlatformIOProf.hpp"
+#include "geopm_debug.hpp"
+#include "geopm/PluginFactory.hpp"
+#include "geopm/PlatformIO.hpp"
+#include "geopm/PlatformTopo.hpp"
+#include "geopm/Helper.hpp"
+#include "geopm/Exception.hpp"
+#include "geopm/Agg.hpp"
+
+namespace geopm
+{
+    const std::map<geopm_domain_e, std::string> FFNetAgent::M_NNET_ENVNAME =
+            {
+                {GEOPM_DOMAIN_PACKAGE, "GEOPM_CPU_NN_PATH"},
+                {GEOPM_DOMAIN_GPU, "GEOPM_GPU_NN_PATH"}
+            };
```

Comment:
```suggestion
    const std::map<geopm_domain_e, std::string> FFNetAgent::M_NNET_ENVNAME =
        {{GEOPM_DOMAIN_PACKAGE, "GEOPM_CPU_NN_PATH"},
         {GEOPM_DOMAIN_GPU, "GEOPM_GPU_NN_PATH"}};
```
Please make this change for all these maps defined up to the constructor.